cmake/nanobind-config.cmake

```diff
@@ -195,7 +195,11 @@ endfunction()
 
 function(nanobind_extension_abi3 name)
   get_filename_component(ext "${NB_SUFFIX}" LAST_EXT)
-  set_target_properties(${name} PROPERTIES PREFIX "" SUFFIX ".abi3${ext}")
+  if(WIN32)
+    set_target_properties(${name} PROPERTIES PREFIX "" SUFFIX ".abi3${ext}")
```

Comment:
I got this exactly backwards. 🤦

Reply:
Going to go ahead and flip these so I can test scikit-build-core, will switch to #222 when it's ready.